espresso-support/core/src/main/java/com/novoda/espresso/AccessibilityViewMatchers.java

```diff
@@ -0,0 +1,187 @@
+package com.novoda.espresso;
+
+import android.content.res.Resources;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+import android.support.annotation.StringRes;
+import android.view.View;
+import android.view.accessibility.AccessibilityNodeInfo;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import static org.hamcrest.Matchers.is;
+
+public class AccessibilityViewMatchers {
+
+    @RequiresApi(Build.VERSION_CODES.LOLLIPOP)
+    public static Matcher<? super View> withUsageHintOnClick(@StringRes final int resourceId) {
+        return new TypeSafeMatcher<View>() {
+
+            private String resourceName = null;
+            private String expectedText = null;
+
+            @Override
+            protected boolean matchesSafely(View view) {
+                if (!view.isClickable()) {
+                    return false;
+                }
+
+                if (null == this.expectedText) {
```

Comment:
commit 253db0766c1bebd86ca112ed8b717833a4280c51
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Mar 28 13:31:06 2017 +0100

    Add missing blank line before end of class